cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/PackCommandTest.java

```diff
@@ -320,6 +320,18 @@ public void testPackEmptyProjectWithCompilerPlugin() throws IOException {
                 getOutput("compile-empty-project-with-compiler-plugin.txt"));
     }
 
+    @Test(description = "Pack an empty package with compiler plugin")
+    public void testPackEmptyProjectWithBuildTools() throws IOException {
+        Path projectPath = this.testResources.resolve("emptyProjectWithBuildTool");
+        System.setProperty("user.dir", projectPath.toString());
```

Comment:
shall we make `user.dir`. as constant?

Reply:
We already have a constant that we can use